pkg/boot/purgatory/purg_load.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:generate go run genpurg.go purgatories.go
+
+// Package purgatory provides two purgatories for use with kexec_load system
```

Comment:
replace two with several, so we are future proof, I think I actually  include 3 now anyway :-)

Reply:
done